samples/basic/main.go

```diff
@@ -0,0 +1,15 @@
+package main
+
+import (
+	"context"
+	"log"
+
+	"github.com/gofrs/uuid"
+	"userclouds.com/authz"
+)
+
+func main() {
+	client := authz.NewClient("http://console-dev.tenant.127.0.0.1.nip.io:3009")
+	obj, err := client.GetObject(context.Background(), uuid.Must(uuid.FromString("087fe0f7-3e3d-4ece-8a61-aea6773d2cd2")))
+	log.Printf("hello, obj = %v, err = %v", obj, err)
```

Comment:
Pretty simple/dumb sample app, I can just write something better before merging this. The UUID here is for the "UserClouds Dev" org which is provisioned in our dev universe.